File: decompiled/3.19.1/be/ehealth/businessconnector/mycarenet/memberdata/builders/ResponseObjectBuilder.java
package be.ehealth.businessconnector.mycarenet.memberdata.builders;

import be.ehealth.businessconnector.mycarenet.memberdata.domain.MemberDataBuilderResponse;
import be.ehealth.technicalconnector.exception.TechnicalConnectorException;
import be.fgov.ehealth.mycarenet.memberdata.protocol.v1.MemberDataConsultationResponse;

public interface ResponseObjectBuilder {
   MemberDataBuilderResponse handleConsultationResponse(MemberDataConsultationResponse var1) throws TechnicalConnectorException;
}

File: src/main/kotlin/org/taktik/connector/business/memberdata/builders/ResponseObjectBuilder.kt
package org.taktik.connector.business.memberdata.builders

import org.taktik.connector.business.memberdata.domain.MemberDataBuilderResponse
import org.taktik.connector.technical.exception.TechnicalConnectorException
import be.fgov.ehealth.mycarenet.memberdata.protocol.v1.MemberDataConsultationResponse
import org.taktik.connector.technical.service.etee.Crypto
import javax.xml.bind.JAXBException
import javax.xml.transform.TransformerConfigurationException

interface ResponseObjectBuilder {
    @Throws(TechnicalConnectorException::class)
    fun handleConsultationResponse(consultResponse: MemberDataConsultationResponse, crypto: Crypto) : MemberDataBuilderResponse?
}
